apps/prairielearn/src/pages/userSettings/userSettings.html.ts

```diff
@@ -77,6 +95,18 @@ export function UserSettings({
             </table>
           </div>
 
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex">Purchases</div>
+
+            ${purchases.length === 0
+              ? html`
+                  <div class="card-body text-muted">
+                    You don&apos;t currently have any purchases.
+                  </div>
+                `
+              : PurchaseTable({ purchases })}
+          </div>
```

Comment:
Possibly move this all into a `UserSettingsPurchasesCard` in the `ee` directory for clean separation. If nothing else, we should only show this card for enterprise instances.

Reply:
Seems reasonable.

Reply:
Moved this all to `ee` in bc26614.